Cargo.lock

```diff
@@ -266,22 +258,23 @@ name = "jmespath"
 version = "0.2.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "lazy_static 0.2.9 (registry+https://github.com/rust-lang/crates.io-index)",
- "serde 1.0.15 (registry+https://github.com/rust-lang/crates.io-index)",
- "serde_json 1.0.3 (registry+https://github.com/rust-lang/crates.io-index)",
+ "lazy_static 0.2.11 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde 1.0.27 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde_json 1.0.9 (registry+https://github.com/rust-lang/crates.io-index)",
  "slug 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
 [[package]]
 name = "jsonapi"
-version = "0.5.2"
+version = "0.5.4"
```

Comment:
with this jsonapi bump, i think we'll need to merge in https://github.com/steveklabnik/rustdoc/pull/232 first, as per this change: https://github.com/michiel/jsonapi-rust/commit/fb66803252dd7866713ce93741548a45ba2596ab